src/radiation/moments.cpp

```diff
@@ -831,11 +831,12 @@ TaskStatus CalculateGeometricSourceImpl(T *rc, T *rc_src) {
         Real dlnalp[ND];
         Real Gamma[ND][ND][ND];
         geom.GradLnAlpha(CellLocation::Cent, iblock, k, j, i, dlnalp);
-        geom.ConnectionCoefficient(CellLocation::Cent, iblock, k, j, i, Gamma);
+        Real dg[ND][ND][ND];
+        geom.MetricDerivative(CellLocation::Cent, k, j, i, dg);
+        Geometry::Utils::SetConnectionCoeffFromMetricDerivs(dg, Gamma);
```

Comment:
What is the reason for this change? I don't see `dg` being used below (but maybe I missed it)

Reply:
nm, I see where it is used below